lib/ansible/module_utils/net_tools/nios/api.py

```diff
@@ -55,25 +54,20 @@
 NIOS_SRV_RECORD = 'record:srv'
 NIOS_NAPTR_RECORD = 'record:naptr'
 NIOS_TXT_RECORD = 'record:txt'
-NIOS_NSGROUP = 'nsgroup'
-NIOS_IPV4_FIXED_ADDRESS = 'fixedaddress'
-NIOS_IPV6_FIXED_ADDRESS = 'ipv6fixedaddress'
-NIOS_NEXT_AVAILABLE_IP = 'func:nextavailableip'
-NIOS_IPV4_NETWORK_CONTAINER = 'networkcontainer'
-NIOS_IPV6_NETWORK_CONTAINER = 'ipv6networkcontainer'
+NIOS_MEMBER = 'member'
 
 NIOS_PROVIDER_SPEC = {
-    'host': dict(fallback=(env_fallback, ['INFOBLOX_HOST'])),
```

Comment:
`env_fallback` module is used when user has defined `INFOBLOX_HOST` value under env variable. Please udo these changes.